nncf/torch/extra_compression_module_type.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+
+class ExtraCompressionModuleType(Enum):
```

Comment:
No need to extract this single enum into an own file, this will be only used along with `NNCFNetwork` and thus can stay in `nncf_network.py`. If importing this from the `nncf_network.py` only causes circular imports in your new debug interface file, just do a local import in the associated function.

Reply:
done